services/web/server/src/simcore_service_webserver/login/_registration.py

```diff
@@ -237,7 +237,7 @@ async def check_and_consume_invitation(
     if confirmation_token := await validate_confirmation_code(invitation_code, db, cfg):
         try:
             invitation = _InvitationValidator.parse_obj(confirmation_token)
-            return invitation.data
+            return cast(InvitationData, invitation.data)
```

Comment:
What about something like this to avoid a cast?
```suggestion
            invitation_data: InvitationData = invitation.data
            return invitation_data
```

Reply:
will correct it in my next PR. thanks :+1: